tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -2177,9 +2178,14 @@ public static function getDrivePermissionsList(
 		string $xRequestId,
 		string $user,
 		string $password,
-		string $spaceId
+		string $spaceId,
+		?string $query = null
 	): ResponseInterface {
 		$url = self::getBetaFullUrl($baseUrl, "drives/$spaceId/root/permissions");
+		if (!empty($query)) {
```

Comment:
```suggestion
		if ($query !== null) {
```

`empty` returns `true` also for `0` `false`